src/Monarch/VirtualDom.ts

```diff
@@ -51,10 +51,10 @@ export interface DiffWork<a, b> {
 
 // prettier-ignore
 interface MkDiffWork {
-    <a>(commitedVNode: VirtualDomTree<a>): <b>(vNode: VirtualDomTree<b>) => DiffWork<a, b>
+    <a, b>(commitedVNode: VirtualDomTree<a>, vNode: VirtualDomTree<b>): DiffWork<a, b>
 }
 
-export const mkDiffWork: MkDiffWork = x => y => ({
+export const mkDiffWork: MkDiffWork = (x, y) => ({
```

Comment:
I want to suggest `mkRootDiffWork` for this function as it is only used for the root virtual DOM tree nodes.

Reply:
Also as a style convention let's write it using the function declaration syntax instead of anonymous function expression syntax.